File: src/GlassVB/GlassVB/Actions/ToggleFullScreenAction.vb
Imports System.Windows
Imports System.Windows.Interactivity

Namespace ESRI.ArcGIS.SilverlightMapApp.Actions
		Public Class ToggleFullScreenAction
			Inherits TriggerAction(Of UIElement)
				Protected Overrides Sub Invoke(ByVal parameter As Object)
						Application.Current.Host.Content.IsFullScreen = Not Application.Current.Host.Content.IsFullScreen
				End Sub
		End Class
End Namespace

File: src/TabbedRibbonCS/TabbedRibbon/Actions/ToggleFullScreenAction.cs
using System.Windows;
using System.Windows.Interactivity;

namespace ESRI.ArcGIS.SilverlightMapApp.Actions
{
    public class ToggleFullScreenAction : TriggerAction<UIElement>
    {
        protected override void Invoke(object parameter)
        {
            Application.Current.Host.Content.IsFullScreen = !Application.Current.Host.Content.IsFullScreen;
        }
    }
}
